tinyagentos/library_collections.py

```
@@ -0,0 +1,362 @@
+            # 4. Link the collection to the library item.
+            try:
+                await http_client.post(
+                    f"{base_url}/collections/{collection_id}/link",
+                    json={"type": "taos", "id": item_id},
+                    headers=auth_headers,
+                )
+                logger.debug(
+                    "Linked collection %s to library item %s",
+                    collection_id, item_id,
+                )
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)